### PR TITLE
Develop (task branch) - common layouts task

### PR DESCRIPTION
**Pull request by Osama-Tahboub1**
Layouts, structure and content has been done for home, portfolio and about pages.
Stylesheet for these files has also been created.

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                    <li><a href="#">Portfolio</a></li>
+                </ul>
+            </nav>
+
+            <h1>My Portfolio</h1>
+        </header>
+
+        <main>
+            <div class="col3">
+                    <h2><a href="/sites/academy-php7/html/Responsive site project/index.html">Pilot Shop</a></h2>
```
Comment (karimkefi) [inline portfolio.html]: remove indentation on row 25 and 26

File: about.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>About Page</title>
+    <link type="text/css" rel="stylesheet" href="normalize.css">
+    <link type="text/css" rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div>
```
Comment (karimkefi) [inline about.html]: do you need this div?

Comment (Osama-Tahboub1) [reply about.html]: No, this is from deleted code, I will remove it

File: portfolio.html (hunk 1 above)
Comment (Osama-Tahboub1) [reply portfolio.html]: This is done.

**Pull request merged**